src/createWavelets.jl

```diff
@@ -99,8 +155,12 @@ function father(c::CWT{<:WaveletBoundary,T}, ω,
     return averaging
 end
 
-
-
+function father(c::CWT{W, T, <:Morse}, ω, averagingType::ContinuousWavelets.Father) where {W,T}
```

Comment:
No need to copy this; because of multiple dispatch, the general version will cover it.

Reply:
The locationShift function for Morse wavelet (cf) is different from the locationShift  for general wavelet (e.g. Morse wavelet (sigma[1])  ).

Reply:
ah, I did miss that. I think it would be more consistent to pass the sWidth parameter on, even if it isn't getting used in `locationShift`, and using the original father wavelet function.